ccs-scale-infra-shared/terraform/modules/infrastructure/network/vpc.tf

```diff
@@ -314,6 +314,16 @@ resource "aws_network_acl" "scale_external" {
     to_port    = 9200
   }
 
+  # Allow outbound FTP (cnet) traffic on port 21
+  egress {
```

Comment:
Assuming the requests to CNET are being initiated by Spree/Sidekiq, those components sit in the private subnets - therefore need two additional rules (following pattern of existing sendgrid / logit.io rules): 

- allow egress from `aws_network_acl.scale_internal` on port 21 to `0.0.0.0/0` - this will route through the NAT Gateway in the public subnets, therefore..
- allow ingress to `aws_network_acl.scale_external` on port 21 from `data.aws_vpc.scale.cidr_block`